.appveyor.yml

```diff
@@ -8,9 +8,8 @@ environment:
   GOPATH: C:\gopath
   PATH: C:\go\bin;C:\tools\mingw64\bin;$(PATH)
   CGO_ENABLED: 1
-  GO111MODULE: off
   matrix:
-    - GO_VERSION: 1.13.12
+    - GO_VERSION: 1.14.4
```

Comment:
For consideration... runc is currently testing with go 1.13 and go 1.14 but only using go 1.14 for vendor  verification. Wondering if we need to go that route as well.